### PR TITLE
Download curation data

### PR DESCRIPTION
**Pull request by prakhar-s**
1)Added validated by and curated by feature.
2)Download the datasheet depending on the filter chosen.

### CODE REVIEW

File: src/components/pages/text-curation/edit/header.tsx [hunk 1]
```diff
@@ -18,8 +21,24 @@ const Actions = () => {
...
     <Flex gap={4} alignItems="center">
+      {canEdit && (
+        <Button
+          p={5}
+          variant="outline"
+          colorScheme="blue"
+          leftIcon={<DownloadIcon />}
+          onClick={() => handleOnDownload(datasetId, rows.filter.curatedStatus)}
+        >
+          Download
```
Comment (harshzalavadiya) [inline src/components/pages/text-curation/edit/header.tsx]: please put this into translation file

File: src/pages/text-curation/show/[datasetId].tsx [hunk 2]
```diff
@@ -17,14 +24,21 @@ export const getServerSideProps = async (ctx) => {
...
 
   const user = getParsedUser(ctx);
+  const userName = user.name;
+  const canEdit =
+    data.contributors.includes(user.id.toString()) ||
+    hasAccess([Role.Admin], ctx) ||
+    data.validators.includes(user.id.toString());
 
-  const canEdit = data.contributors.includes(user.id.toString()) || hasAccess([Role.Admin], ctx);
+  const canValidate = data.validators.length == 0 || data.validators.includes(user.id.toString());
```
Comment (harshzalavadiya) [inline src/pages/text-curation/show/[datasetId].tsx]: admin check is removed from here, is that intentional?

**Review by harshzalavadiya — CHANGES_REQUESTED** [review]

File: locales/en/text-curation.json [hunk 3]
```diff
@@ -18,6 +18,7 @@
   },
   "dataset_description": "Dataset Description",
   "date_columns": "Date Columns",
+  "Download": "Download",
```
Comment (harshzalavadiya) [inline locales/en/text-curation.json]: @prakhar-s please follow key naming consistency, it has to be lowercase, without space (use underscore instead)

**Review by harshzalavadiya — CHANGES_REQUESTED** [review]

File: src/components/pages/text-curation/edit/header.tsx (hunk 1 above)
Comment (prakhar-s) [reply src/components/pages/text-curation/edit/header.tsx]: done.

File: src/pages/text-curation/show/[datasetId].tsx (hunk 2 above)
Comment (prakhar-s) [reply src/pages/text-curation/show/[datasetId].tsx]: haven't removed admin access, just added the access to validators also

File: locales/en/text-curation.json (hunk 3 above)
Comment (prakhar-s) [reply locales/en/text-curation.json]: done

**Pull request merged**